vendor/bat-native-ledger/src/bat/ledger/internal/wallet/create.cc

```diff
@@ -31,35 +31,7 @@ Create::Create(bat_ledger::LedgerImpl* ledger) :
 Create::~Create() {
 }
 
-void Create::Start(const std::string& safetynet_token,
-      ledger::CreateWalletCallback callback) {
-  if (!safetynet_token.empty()) {
-    std::string safetynet_prefix = PREFIX_V5;
-#if defined (OS_ANDROID) && defined(ARCH_CPU_X86_FAMILY)\
-    && defined(OFFICIAL_BUILD)
-    safetynet_prefix = PREFIX_V3;
```

Comment:
is it fine that we use a common prefix for x86 and arm right now?

Reply:
That prefix was used for SafetyNet check only. With checking on client side we don't need it anymore.